### ISSUE TITLE
feat: Permissionless CL Pool "Nice To Haves"

### ISSUE THREAD

**Issue by czarcas7ic** [osmosis-labs/osmosis#6396]
## Background

- [ ] Programatic ranked quote asset order
- [x] Restrict sig fig precision of quote assets
- [ ] Lower min spot price
- [x] Restrict set of allowed spread factors

## Suggested Design

TBD

## Acceptance Criteria

TBD

**Comment by ValarDragon** [comment]
As discussed in slack, I consider all of these as nice to haves personally, and not blockers for governance

**Comment by czarcas7ic** [comment]
Agreed, will change the name but will keep this open so we are still tracking these items

**Comment by ValarDragon** [comment]
>  Programatic ranked quote asset order

This is done right?

**Comment by czarcas7ic** [comment]
@ValarDragon I don't see anything in the code that orders denoms based on any kind of ranking. All I see is validation that denom1 is in fact a authorizedQuoteDenom

https://github.com/osmosis-labs/osmosis/blob/ed96fa9066a8e5dc6604b3e3c081366ce8d49e48/x/concentrated-liquidity/pool.go#L365-L372
